habitat_sim/physics.py

```diff
@@ -36,6 +36,7 @@
     "VelocityControl",
     "RayHitInfo",
     "RaycastResults",
+    "ContactPointData",
```

Comment:
Are you on master? This should already be fixed in master in #1372

Reply:
```suggestion
```

Reply:
Fixed.